app/views/qualified_opportunities/index.haml

```diff
@@ -7,7 +7,7 @@
     %p
       #{@client.full_name} is already active in at least one match:
       - @client.active_matches.each do |match|
-        %br
+        %p
```

Comment:
This is fine, but I actually think the issue is just that the `=` on the line below the `%br` is indented.  Can you test and see if just outdenting that to the same depth as the %br fixes it? (Adding p tags seems extreme and may bring unwanted padding along for the ride.)

Reply:
Outdenting doesn't help. If I remove the p-tag in line 7 then it does. Everything then gets centered, though.

Reply:
Thanks for testing.  p tag it is!